src/Configuration/src/ConfigServer/ConfigServerClientSettings.cs

```diff
@@ -114,6 +114,16 @@ public sealed class ConfigServerClientSettings
     /// </summary>
     public const bool DefaultHealthEnabled = true;
 
+    /// <summary>
+    /// Default Encryption method.
+    /// </summary>
+    public const string DefaultÈncryptionRsaAlgorithm = "DEFAULT";
```

Comment:
I suspect the accent on `È` wasn't intentional.

Reply:
fixed